tensorflow/lite/micro/kernels/svdf_test.cc

```diff
@@ -466,6 +466,7 @@ const float golden_output_16x1x1[] = {
     -0.603093, 0.970736,  -3.567897, 0.035085,  -0.201711, -0.550400, 1.545573,
     -1.805005};
 
+#if !defined(XTENSA)  // Needed to avoid build errors from unsused variables.
```

Comment:
See comment below. I think we should be using this (and it was an error on my part that we were not).

Reply:
done